File: app/src/main/java/com/dhruvdroid/jetpack/HomeActivity.java
package com.dhruvdroid.jetpack;
//
// Created by  on 09/07/20.
//

import android.app.Activity;
import android.os.Bundle;

import androidx.annotation.Nullable;

import com.dhruv_droid.jetpack.R;

public class HomeActivity extends Activity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_home);
    }
}

File: app/src/main/java/com/dhruvdroid/jetpack/HomeActivity.kt
package com.dhruvdroid.jetpack

import android.os.Bundle
import android.os.PersistableBundle
import androidx.appcompat.app.AppCompatActivity
import com.dhruv_droid.jetpack.R
import dagger.hilt.android.AndroidEntryPoint

//
// Created by  on 09/07/20.
//
@AndroidEntryPoint
class HomeActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_home)
    }

    override fun onCreate(savedInstanceState: Bundle?, persistentState: PersistableBundle?) {
        super.onCreate(savedInstanceState, persistentState)
    }
}